x-pack/test/security_api_integration/tests/session_idle/cleanup.ts

```diff
@@ -30,7 +30,9 @@ export default function ({ getService }: FtrProviderContext) {
   }
 
   async function getNumberOfSessionDocuments() {
-    return (await es.search({ index: '.kibana_security_session*' })).hits.total.value;
+    return (((await es.search({ index: '.kibana_security_session*' })).hits.total as unknown) as {
```

Comment:
__note:__ typechecker suddenly started to complain about these return types (types are invalid indeed).